common/persistence/serialization/task_serializer.go

```diff
@@ -1050,6 +1051,7 @@ func (s *TaskSerializer) visibilityDeleteTaskFromProto(
 		VisibilityTimestamp:            deleteVisibilityTask.VisibilityTime.AsTime(),
 		TaskID:                         deleteVisibilityTask.TaskId,
 		CloseExecutionVisibilityTaskID: deleteVisibilityTask.CloseVisibilityTaskId,
+		WorkflowCloseTime:              deleteVisibilityTask.CloseTime.AsTime(),
```

Comment:
Why here `WorkflowCloseTime` and not `CloseTime`?

Reply:
There wasn't any specific reason I named it `WorkflowCloseTime`, I think it was because it has field `WorkflowKey` and I tried to match that.

Reply:
Change to `CloseTime` to keep consistent please.